File: app/src/main/java/com/akash/audiowidget/MyWidgetRemoteViewsService.java
package com.akash.audiowidget;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class MyWidgetRemoteViewsService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new WidgetRemoveViewFactory(getApplicationContext(), intent);
    }
}

File: app/src/main/java/com/akash/audiowidget/MyWidgetRemoteViewsService.kt
package com.akash.audiowidget

import android.content.Intent
import android.widget.RemoteViewsService

class MyWidgetRemoteViewsService : RemoteViewsService() {

    override fun onGetViewFactory(intent: Intent): RemoteViewsService.RemoteViewsFactory {
        return WidgetRemoveViewFactory(applicationContext, intent)
    }
}
